### PR TITLE
Devel/v1.0

### PR DESCRIPTION
**Pull request by mariojmdavid**
first release of the baseline document
v1.0

### CODE REVIEW

File: content/03.intro.md [hunk 1]
```diff
@@ -42,21 +42,21 @@ requirements.
 
 ## Contextualization of a Service
 
-As a result, a **Service**, as conceived in this document, represent the following:
+As a result, a **Service**, as conceived in this document, represents the following:
 
-* Web service [WS] [@url:https://techterms.com/definition/web_service]:
+* **Web service** [WS] [@url:https://techterms.com/definition/web_service]:
```
Comment (orviz) [inline content/03.intro.md:47]: I think we can safely remove the `[WS]`, `[WApp]` and `[Plat]` acronyms, throughout the doc we are using the full-length version e.g. **Service** or **Service Composition**

File: content/03.intro.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The meaning of **Service** can be regarded from different perspectives.
 From an IT Service Management (ITSM) standpoint, such as the EOSC Service Management
```
Comment (orviz) [inline content/03.intro.md:12]: Add `ITSM` and `SMS` to acronyms

File: content/03.intro.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ The meaning of **Service** can be regarded from different perspectives.
...
 System (SMS) process model, a service is devised as a means to "provide value
```
Comment (orviz) [inline content/03.intro.md:13]: Should 'service' be in bold (**Service**) here? Same applies to the rest 'service' mentions in the Intro

File: content/03.intro.md [hunk 4]
```diff
@@ -42,21 +42,21 @@ requirements.
...
-    * A web application or "web app" is a software program that runs on a web server.
-    * Web apps must be accessed through a web browser.
+* **Web application** [WApp] [@url:https://techterms.com/definition/web_application]:
+    * A web application or "web app" is a software program that is delivered over
+    the Internet and is accessed through a web browser.
 
-* Platform or Service Composition [Plat]
+* **Platform** or **Service Composition** [Plat]
 [@url:https://csrc.nist.gov/glossary/term/Service_Composition]:
     * Aggregation of multiple small services into larger services.
```
Comment (orviz) [inline content/03.intro.md:61]: I would add here a reference to common service architectures (indeed microservices are mentioned in 06.quality-criteria.md), sth like:

`Aggregation of multiple small services into larger services, according to a service-oriented (SOA) and/or microservices architecture.`

If so, we need to add `SOA` to acronym list

File: content/06.quality_criteria.md [hunk 5]
```diff
@@ -8,50 +8,123 @@ and operation process within the framework of the EOSC-Synergy project.
...
 Integration testing refers to the evaluation of the interactions among
-coupled **Service** or parts of a system that cooperate to achieve a given
+coupled **Services** or parts of a system that cooperate to achieve a given
 functionality.
 
-* **[SvcQC.Int01]** Integration testing outcome MUST guarantee the overall
-operation of the  **Service** whenever new functionality is involved. [SQA-QC.Int01].
+* **[SvcQC.Int01]** Whenever a new functionality is involved, integration
+testing MUST guarantee the operation of any previously-working interaction
+with external **Services**. [SQA-QC.Int01].
```
Comment (orviz) [inline content/06.quality_criteria.md:55]: Should we add in this version links to the specific criteria in the SQA online document?

File: content/06.quality_criteria.md [hunk 6]
```diff
@@ -90,53 +163,116 @@ The identified types of documentation and their RECOMMENDED content are:
...
+  * **[SvcQC.Sec06.3]** The security assessment of the target system 
+  configuration is particularly important to reduce the risk of security
+  attacks. The benchmarks delivered by the Center for Internet
+  Security (CIS) [@url:https://www.cisecurity.org/cis-benchmarks/] and the
+  NIST's Security Assurance Requirements for Linux Application Container
+  Deployments [@url:https://doi.org/10.6028/NIST.IR.8176] MUST be considered
+  for this task.
+
+* **[SvcQC.Sec07]** IaC testing, from [SvcQC.Aud02] criterion, MUST cover the
+security auditing of the IaC templates (aka _Security as Code_ or _SaC_) in
```
Comment (orviz) [inline content/06.quality_criteria.md:232]: Can use `SaC` as it was defined in the section intro

File: content/06.quality_criteria.md [hunk 7]
```diff
@@ -90,53 +163,116 @@ The identified types of documentation and their RECOMMENDED content are:
...
+  non-encrypted secrets or disabled audit logs.
+  * **[SvcQC.Sec07.4]** SaC MAY be used to seek, in the IaC templates, for
+  violations of security requirements outlined in the applicable regulations
+  from criterion [SvcQC.Sec05].
 
-* **[SvcQC.Sec07]** The **Service** MUST validate the credentials and signatures.
+### Policies [SvcQC.Pol]
 
-### Automated Deployment [SvcQC.Aud]
+Policy documents describe what are the users expected behaviour when using the
```
Comment (orviz) [inline content/06.quality_criteria.md:249]: users -> user's

File: content/06.quality_criteria.md [hunk 8]
```diff
@@ -149,13 +285,86 @@ Level Agreement (OLA) with the infrastructure where it is integrated.
...
-
-* **[SvcQC.MAM01]** The **Service** in an operational production state SHOULD be monitored.
-  * **[SvcQC.MAM01.1]** The **Service** public endpoints SHOULD be monitored.
-  * **[SvcQC.MAM01.2]** The **Service** public APIs SHOULD be monitored.
-  * **[SvcQC.MAM01.3]** The **Service** SHOULD for security related criteria (DAST).
+### Monitoring [SvcQC.Mon]
+
+Monitoring is a periodic testing of the **Service**. It requires a monitoring
+service from where tests are executed or sent and results of those tests are shown.
+The tests can be the same, in part or in total of the Functional tests.
```
Comment (orviz) [inline content/06.quality_criteria.md:292]: I would add here the other 2 types identified below: security and infrastructure

File: content/03.intro.md [hunk 9]
```diff
@@ -42,21 +42,21 @@ requirements.
...
     * Web services are designed to communicate with other programs,
     rather than directly with users.
     * Most web services provide an API, or a set of functions and commands,
     that can be used to access the data.
 
-* Web application [WApp] [@url:https://techterms.com/definition/web_application]:
-    * A web application or "web app" is a software program that runs on a web server.
-    * Web apps must be accessed through a web browser.
+* **Web application** [WApp] [@url:https://techterms.com/definition/web_application]:
+    * A web application or "web app" is a software program that is delivered over
```
Comment (orviz) [inline content/03.intro.md:56]: we use `website` a lot in the 06.quality-criteria.md, maybe we should also include it here

File: content/03.intro.md (hunk 1 above)
Comment (mariojmdavid) [reply content/03.intro.md:47]: done

File: content/03.intro.md (hunk 2 above)
Comment (mariojmdavid) [reply content/03.intro.md:12]: done

File: content/03.intro.md (hunk 3 above)
Comment (mariojmdavid) [reply content/03.intro.md:13]: agree, done

File: content/03.intro.md (hunk 4 above)
Comment (mariojmdavid) [reply content/03.intro.md:61]: done
also added SOA to acronym list

File: content/06.quality_criteria.md (hunk 5 above)
Comment (mariojmdavid) [reply content/06.quality_criteria.md:55]: I think so, gives some link and/or background, but probably up for some discussion
I will let the reference for now.

File: content/06.quality_criteria.md (hunk 6 above)
Comment (mariojmdavid) [reply content/06.quality_criteria.md:232]: done

File: content/06.quality_criteria.md (hunk 7 above)
Comment (mariojmdavid) [reply content/06.quality_criteria.md:249]: done

File: content/06.quality_criteria.md (hunk 8 above)
Comment (mariojmdavid) [reply content/06.quality_criteria.md:292]: correct
but realized Monitoring should be after Automated Deployment
moved accordingly

File: content/03.intro.md (hunk 9 above)
Comment (mariojmdavid) [reply content/03.intro.md:56]: only in [SvcQC.Pol01]
I think better to change "website" to **Service**

**Pull request merged**